src/features/bundlr/BundlrInput.tsx

```diff
@@ -0,0 +1,36 @@
+import {
+  NumberInput,
+  NumberInputField,
+  NumberInputStepper,
+  NumberIncrementStepper,
+  NumberDecrementStepper,
+  NumberInputProps,
+} from '@chakra-ui/react';
+
+export function BundlrInput(props: NumberInputProps) {
+  return (
+    <NumberInput
```

Comment:
I'm not sure I agree with a number input field here. At least the steppers, there are 18 decimals places and lot of uploads are going to be very small numbers. Steppers would make sense for an integer. I realize this was in the designs I can bring with them also.

Reply:
I don't quite care for the steppers either. I don't see much of a use case for them.